README.md

```diff
@@ -18,7 +18,7 @@ Open-source code for paper: "Direct Multiple Shooting for Computationally Effici
 
 - Activate the virtual environment: `.\.env\Scripts\activate`
 
-- Install dependencies with: `pip install -r requirements.txt`
+- Install dependencies with: `pip install -e .`
```

Comment:
Not "dependencies" any more. I would say "Install package" (please also double-check that we indicate the right directory to run the command from).